index.html

```diff
@@ -2085,14 +2097,17 @@ <h1>Speakers</h1>
                                             <a target="default" href="www.mozillaphilippines.org"><i class="icon fa fa-external-link"></i></a>
                                             <a target="default" href="https://twitter.com/bobreyes"><i class="icon social_twitter"></i></a>
                                             <a target="default" href="https://www.linkedin.com/in/bobreyes/"><i class="icon social_linkedin"></i></a>
+                                            <a target="default" href="https://www.facebook.com/robertbobreyes/
+
+                                            "><i class="icon fa fa-facebook"></i></a>
```

Comment:
Some part of above anchor tag is coming here. Kindly fix it. `">` the anchor tag should close on line 2100. Try to write this line as it is same in line no 2098, 2099.